src/structs.md

```diff
@@ -17,3 +17,17 @@ fn main() {
     println!("{} is {} years old", peter.name, peter.age);
 }
 ```
+
+<details>
+
+* The top of the program is how we define a struct, the main function creates a struct, and then at the bottom we access the fields.
+* Structs work similar as they do in other languages, but note there is also no typedef keyword or inheritance between structs. Also note there are no functions. The implementation is separate.
+* We can access the structs fields with the dot notation 
+* Using short-hand syntax, we can initiate the fields with similar-named variables.  [define age variable on it’s own line] 
```

Comment:
nit: "its own line"

Reply:
Right, and we don't have a convention of putting things into `[ ... ]` brackets.